gammapy/stats/counts_statistic.py

```diff
@@ -234,13 +234,13 @@ def error(self):
 
     @property
     def TS_null(self):
-        """Stat value for null hypothesis, i.e. 0 expected signal counts"""
-        return wstat(self.n_on, self.n_off, self.alpha, 0)
+        """Stat value for null hypothesis, i.e. mu_sig expected signal counts"""
+        return wstat(self.n_on, self.n_off, self.alpha, self.mu_sig)
 
     @property
     def TS_max(self):
-        """Stat value for best fit hypothesis, i.e. expected signal mu = n_on - alpha * n_off"""
-        return wstat(self.n_on, self.n_off, self.alpha, self.excess)
+        """Stat value for best fit hypothesis, i.e. expected signal mu = n_on - alpha * n_off - mu_sig"""
+        return wstat(self.n_on, self.n_off, self.alpha, self.excess + self.mu_sig)
 
     def _stat_fcn(self, mu, delta=0, index=None):
         return wstat(self.n_on[index], self.n_off[index], self.alpha[index], mu) - delta
```

Comment:
shouldn't you change this as well to take into account `self.mu_sig`? Otherwise the error computation will be incorrect no?
see https://gist.github.com/registerrier/b3dc6dc41349ae466de8a00af3d748fb